lib/index.js

```diff
@@ -553,4 +554,93 @@ function equals(mat, operand) {
     }
 
     return op1.every((val, ind1) => val.every((ele, ind2) => Math.abs(ele - op2[ind1][ind2]) < 1e-10));
-}   
+}
+
+
+/**
+ * Calculates the adjunct matrix
+ *
+ * @param mat input matrix that initialized the function
+ * @returns adjunct matrix
+ */
+
+ function adjunct(mat) {
```

Comment:
Isn't it called adjoin?

Reply:
Also, adjoin is mainly used to compute the determinant. However, I'm able to compute it using row elimination. What else it is useful for?